tests/foreman/longrun/test_inc_updates.py

```diff
@@ -49,8 +50,13 @@ def setUpClass(cls):
         ).create()
 
         # Step 3: Upload manifest
-        with manifests.clone() as manifest:
-            upload_manifest(cls.org.id, manifest.content)
+        try:
+            # Update timeout to 10 minutes to finish manifest upload
+            entity_mixins.TASK_TIMEOUT = 600
+            with manifests.clone() as manifest:
+                upload_manifest(cls.org.id, manifest.content)
+        finally:
+            entity_mixins.TASK_TIMEOUT = cls.old_task_timeout
```

Comment:
If there is more use in future, this will be a good case for the use of a context manager

```python
from robottelo.tools import set_in_context

with set_in_context({entity_mixins: ('TASK_TIMEOUT': 600)}):
    with manifest.clone() as manifest:
        upload_manifest(cls.org.id, manifest.content)
```

The implementation of set_in_context could be something like this (need improvements)

```python
from contextlib import contextmanager

@contextmanager
def set_in_context(d):
    old_values = defaultdict({})
    # store old values
    for key, value in d.items():
        old_values[key][value[0]] = getattr(key, value[0])
        setattr(key, value[0], value[1])
    
    yield

    # set the values back to original
    for key, value in old_values.items():
        setattr(key, value[0], value[1])
```

maybe there is already some python lib that does that job, I'll try to find one